pkg/util/tests/cmd.go

```diff
@@ -0,0 +1,80 @@
+package tests
+
+import (
+	"bufio"
+	"os"
+	"os/exec"
+	"os/user"
+	"strings"
+)
+
+func findK3sExecutable() string {
+	k3sBin := "dist/artifacts/k3s"
+	for {
+		_, err := os.Stat(k3sBin)
+		if err != nil {
+			k3sBin = "../" + k3sBin
+			continue
+		}
+		break
+	}
+	return k3sBin
+}
+
+func IsRoot() bool {
+	currentUser, err := user.Current()
+	if err != nil {
+		return false
+	}
+	return currentUser.Username == "root"
+}
+
+func K3sCmd(cmdName string, cmdArgs ...string) (string, error) {
+	k3sBin := findK3sExecutable()
+	// Only run sudo if not root
+	var cmd *exec.Cmd
+	if IsRoot() {
+		k3sCmd := append([]string{cmdName}, cmdArgs...)
+		cmd = exec.Command(k3sBin, k3sCmd...)
+	} else {
+		k3sCmd := append([]string{k3sBin, cmdName}, cmdArgs...)
+		cmd = exec.Command("sudo", k3sCmd...)
+	}
+	byteOut, err := cmd.CombinedOutput()
+	return string(byteOut), err
+}
+
+func FindStringInCmdAsync(scanner *bufio.Scanner, target string) bool {
+	for scanner.Scan() {
+		if strings.Contains(scanner.Text(), target) {
+			return true
+		}
+	}
+	return false
+}
+
+//Launch a k3s command asynchronously
```

Comment:
Can you update this to the standard doc comment format and include one for the new, exported functions?

Reply:
Comments added.